components/servo/servo.go

```diff
@@ -31,6 +31,30 @@ const SubtypeName = "servo"
 var API = resource.APINamespaceRDK.WithComponentType(SubtypeName)
 
 // A Servo represents a physical servo connected to a board.
+//
+// Move example:
+//    // Move the servo from its origin to the desired angle of 30 degrees.
+//    myServoComponent.Move(context.Background(), 30, nil)
+//
+// Position example:
+//    // Get the current set angle of the servo.
+//    pos1, err := myServoComponent.Position(context.Background(), nil)
+//
+//    // Move the servo from its origin to the desired angle of 20 degrees.
+//    myServoComponent.Move(context.Background(), 20, nil)
+//
+//    // Get the current set angle of the servo.
+//    pos2, err := myServoComponent.Position(context.Background(), nil)
+//
+//    logger.Info("Position 1: ", pos1)
+//    logger.Info("Position 2: ", pos2)
+//
+// Stop example: (from the resource.Actuator interface)
```

Comment:
@andf-viam How should we handle this for automation? It's in the docs for servo, but not in the interface here.

Reply:
Right now we are manually populating inherited content for resources that inherit (i.e. Servo (inherits from Actuator) > Stop. In the future we will automate scraping of the inherited class directly (i.e. scrape Actuator directly).

In both cases, I would recommend you omit all code samples for inherited functions. (example: there's nowhere to force `Stop` appearing in the [upstream GO SDK docs ](https://pkg.go.dev/go.viam.com/rdk@v0.26.0/components/servo)anyways.)

Reply:
I can remove stop; I just added it because it's in the docs page for the servo API. When you say

> There's nowhere to force `stop` appearing in the upstream Go SDK docs

do you mean that though my example here would appear in the Go SDK docs, Stop itself doesn't appear in the interface on this page? I am aware of this, just trying to make sure I understand.

Reply:
Sure: https://pkg.go.dev/go.viam.com/rdk@v0.26.0/resource#Actuator is where the `Stop` code sample should go, the scraper will get it from there. Similarly, `Close` code sample should go here: https://pkg.go.dev/go.viam.com/rdk@v0.26.0/resource#Resource . 👍